applications/TrilinosApplication/custom_python/add_trilinos_strategies_to_python.cpp

```diff
@@ -139,12 +139,22 @@ void AddStrategies(pybind11::module& m)
 
     typedef ResidualBasedLinearStrategy< TrilinosSparseSpaceType, TrilinosLocalSpaceType, TrilinosLinearSolverType> TrilinosLinearStrategy;
     py::class_< TrilinosLinearStrategy , typename TrilinosLinearStrategy::Pointer, TrilinosBaseSolvingStrategyType >
-    (m,"TrilinosLinearStrategy").def(py::init< ModelPart&, TrilinosBaseSchemeType::Pointer, TrilinosLinearSolverType::Pointer, TrilinosBuilderAndSolverType::Pointer, bool, bool, bool, bool >())
+    (m,"TrilinosLinearStrategy")
+        .def(py::init([](ModelPart& rModelPart, TrilinosBaseSchemeType::Pointer pScheme, TrilinosLinearSolverType::Pointer pLinearSolver, TrilinosBuilderAndSolverType::Pointer pBuilderAndSolver, bool CalculateReactionFlag, bool ReformDofSetAtEachStep, bool CalculateNormDxFlag, bool MoveMeshFlag) {
+            KRATOS_WARNING("TrilinosLinearStrategy") << "Using deprecated constructor. Please use constructor without linear solver.";
```

Comment:
As this is used in a few places, I'd directly change the occurrences and remove the deprecated constructor exposure.

Reply:
I am little bit concerned on removing it because, may be someone else is using this constructor in a branch, and this will break it. So I was thinking to have the warning for some time and then remove it (at the same time c++ dprecated ctors are removed). What do you say?

Reply:
furthermore code that uses trilinos is still very little tested so it is easy to break